src/Files.App/Views/LayoutModes/ColumnViewBrowser.xaml.cs

```diff
@@ -205,6 +205,52 @@ public void NavigateUp()
 			}
 		}
 
+		public void MoveFocusToPreviousBlade(int currentBladeIndex)
+		{
+			if (currentBladeIndex <= 0)
+				return;
+
+			DismissOtherBlades(currentBladeIndex);
+
+			var activeBlade = ColumnHost.ActiveBlades[currentBladeIndex - 1];
+			activeBlade.Focus(FocusState.Programmatic);
+
+			var activeBladeColumnViewBase = RetrieveBladeColumnViewBase(activeBlade);
+			if (activeBladeColumnViewBase == null)
+				return;
```

Comment:
Can you change `==` to `is`? We have just changed this throughout the whole code base (#10326)